.github/workflows/ci.yml

```
@@ -125,3 +121,14 @@ jobs:
       - name: Whitespace
         run: |
           .ci/test_whitespace.sh
+
+  documentation:
+    name: Documentation clash_version 9.6.4
```

```suggestion
    name: Documentation
```

9.6.4 => GHC version. But also, it's fine to not mention it in the CI job name anyway.

---

Also also, why are we setting Clash's version exactly?

---

Good question, I'm removing it entirely